### PR TITLE
#201: Merge "Systems: Beginner" to training

### PR DESCRIPTION
**Pull request by ga-hegsvold**
I've added comments to some of the sections. Would like your opinion on those.

### CODE REVIEW

File: training/020_systems/020_Beginner.rst [hunk 1]
```diff
@@ -7,62 +7,240 @@ Systems: Beginner
...
 
-Introduction to SQL & Json systems
+Introduction to SQL & JSON systems
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+No such thing as a JSON system. We do have a JSON *source*.
+
+Should we even explain specific systems here?
+Or would it be sufficient to reference the docs?
+
```
Comment (JonasAls89) [inline training/020_systems/020_Beginner.rst:52]: Think we can omit this yes.

File: training/020_systems/020_Beginner.rst [hunk 2]
```diff
@@ -7,62 +7,240 @@ Systems: Beginner
...
+.. _practice-system-config-initial:
+.. code-block:: json
+  :caption: Practice system config - initial
+  :linenos:
+
+  {
+    "_id": "difi",
+    "type": "system:url",
+    "url_pattern": "https://ws.geonorge.no/kommuneinfo/v1/%s"
```
Comment (JonasAls89) [inline training/020_systems/020_Beginner.rst:130]: Du mangler komma "," etter "url_pattern"

File: training/020_systems/020_Beginner.rst [hunk 3]
```diff
@@ -7,62 +7,240 @@ Systems: Beginner
...
+
+.. _practice-system-config-env-var-ref:
+.. code-block:: json
+  :caption: Practice system config with environment variable reference
+  :linenos:
+
+  {
+    "_id": "difi",
+    "type": "system:url",
+    "url_pattern": "$ENV(difi-api)"
```
Comment (JonasAls89) [inline training/020_systems/020_Beginner.rst:229]: "," komma igjen

File: training/020_systems/020_Beginner.rst [hunk 4]
```diff
@@ -71,6 +249,9 @@ JSON Push & Pull protocol
 
 Lots of info in docs.
 
+Yes, so should we even write about it here?
+Or should we just reference the docs?
```
Comment (JonasAls89) [inline training/020_systems/020_Beginner.rst:253]: Ref the docs.

**Review by JonasAls89 — CHANGES_REQUESTED** [review]

File: training/020_systems/020_Beginner.rst (hunk 2 above)
Comment (ga-hegsvold) [reply training/020_systems/020_Beginner.rst]: fixed

File: training/020_systems/020_Beginner.rst (hunk 3 above)
Comment (ga-hegsvold) [reply training/020_systems/020_Beginner.rst]: fixed

**Review by JonasAls89 — APPROVED** [review]

**Review by gabriellsesam — APPROVED** [review]
See latest commit for my changes

**Pull request merged**